.github/templates/bazel_ci_workflow.yml.j2

```diff
@@ -94,7 +94,7 @@ on:
           export COMMIT_TIME
           pip3 install requests==2.26
           python3 -m tools.stats.upload_binary_size_to_scuba || exit 0
-      - name: Test
+      - name: Test ${{ github.workflow }}
```

Comment:
we just consolidated the `Build` and `Test` steps to have a single name (makes tracking them all in one query much easier), if this is just for logs could we instead put this in an env (github may already do this by default) and `echo` it in the `run:` script below?

Reply:
When steps fails, name of the step would be the very first line in GitHub UI.
I though step-names are UX feature only and all logging should be based on job names, shouldn't it?